.zuul.d/jobs.yaml

```diff
@@ -2,7 +2,7 @@
 - job:
     name: ansible-builder-tox-integration
     parent: ansible-buildset-registry-consumer
-    timeout: 3600
+    timeout: 4500
```

Comment:
I actually think we need to set timeout in the job below too. Which is surprising, we might be hitting a bug here.